app/helpers/reports_helper.rb

```diff
@@ -108,4 +108,56 @@ def self.fake_genus(tax_id, tax_info)
     tax_info[:genus_tax_id] = fake_genus_id
     fake_genus_info
   end
+
+  def report_csv_from_params_v2(sample, params)
+    background_id = params[:background_id] || sample.default_background_id
+    background_id = background_id.to_i
+    pipeline_run = select_pipeline_run(sample, params[:pipeline_version])
+    pipeline_run_id = pipeline_run ? pipeline_run.id : nil
+    return "" if pipeline_run_id.nil? || pipeline_run.total_reads.nil? || pipeline_run.adjusted_remaining_reads.nil?
```

Comment:
We do not do this verification on report_v2 - should we? IN any case we should probably return an error in this case.

Reply:
Added the check and error logs to the beginning of report generation.